Tests-Legacy/L0/ANT/_suite.ts

```diff
@@ -27,7 +27,7 @@ describe('ANT Suite', function() {
     it('run ANT with all inputs', (done) => {
         setResponseFile('antGood.json');
 
-        var tr = new trm.TaskRunner('Ant');
+        var tr = new trm.TaskRunner('ANT');
```

Comment:
We are creating the output folder with the source folder name not the task.json name any more. Fixed failing case sensitive tests on Linux.